docs/user/OAuth2OIDC-keycloak.md

```diff
@@ -0,0 +1,7 @@
+# Keycloak
+
```

Comment:
ditto

Reply:
Agree, let me reorder that and include some extra info to the headline so if you land directly to that page, you can get some context.